installazione_archlinux_markdown.md

```diff
@@ -1307,24 +1322,45 @@ Seguirà un elenco di comandi che include software o trick che normalmente appli
 >
 > verrà indicato yay come p.m. per aur, ma potete ovviamente usare il package manager da voi installato  
 
+### paru
+
 ```bash
 #media tool
-yay -S clementine audacity vlc gimp gst-plugins-good gst-plugins-bad gst-plugins-ugly gst-plugins-base gst-libav gvfs alsa-firmware alsa-lib alsa-oss alsa-utils pulseaudio-alsa pavucontrol 
+paru -S ffmpeg mpv clementine audacity vlc gimp gst-plugins-good gst-plugins-bad gst-plugins-ugly gst-plugins-base gst-libav gvfs alsa-firmware alsa-lib alsa-oss alsa-utils pulseaudio-alsa pavucontrol 
 
 #work tool
-yay -S visual-studio-code-bin jdk8-openjdk jdk-openjdk jre-openjdk jre8-openjdk openjdk-doc openjdk-src openjdk8-doc openjdk8-src terminator octave xterm libreoffice-fresh libreoffice-fresh-it hyphen-it mythes-it hunspell-it texlive-bin texlive-core texlive-bibtexextra texlive-fontsextra texlive-formatsextra texlive-games texlive-humanities texlive-latexextra texlive-music texlive-pictures texlive-pstricks texlive-publishers texlive-science texstudio python-pip mariadb 
+paru -S visual-studio-code-bin jdk8-openjdk jdk-openjdk jre-openjdk jre8-openjdk openjdk-doc openjdk-src openjdk8-doc openjdk8-src terminator octave xterm libreoffice-fresh libreoffice-fresh-it hyphen-it mythes-it hunspell-it texlive-bin texlive-core texlive-bibtexextra texlive-fontsextra texlive-formatsextra texlive-games texlive-humanities texlive-latexextra texlive-music texlive-pictures texlive-pstricks texlive-publishers texlive-science texstudio python-pip mariadb 
 
 #net tool
-yay -S mailspring firefox firefox-i18n-it thunderbird thunderbird-i18n-it deluge google-chrome
+paru -S mailspring firefox firefox-i18n-it thunderbird thunderbird-i18n-it deluge chromium
```

Comment:
aggiungere un commento più che cambiare radicalmente google chrome con la sua alternativa open.

Reply:
Non ironicamente farei installare `firefox` e aggiungerei una nota per `chrome` e `chromium`.

Essendo Firefox di base su tutte o quasi tutte le distribuzioni Linux, lo vedo quasi uno standard. In alternativa almeno citarlo.

Reply:
la sezione del **comandone** è una sezione che contiene una serie di programmi che voglio io ogni installazione
come descritto dal suo commento iniziale:
> A voi il compito, se siete curiosi, di informarvi su cosa servono e perché dovreste o non dovreste installarli!   
Spetta all'utente finale se vuoi leggersi e decidere quali volere o non volere. 

mi immagino della guida forkata da te il comandone esattamente con firefox ahahahah tuttavia quello è un copy-pasta che serve più che altro a me ad installare una serie di cose che uso in genere nelle mie distro ( tra cui c'è sia firefox che google-chrome. Anzi oggi dovrei aggiornarla inserendo più che firefox che non uso più **vivaldi** )

Quella sezione non va "toccata" di base perché non fa parte della guida in realtà, ma è più un promemoria personale